### ISSUE TITLE
Quality report test on several top repositories

### ISSUE THREAD

**Issue by EgorBu** [src-d/style-analyzer#311]
Create a long-running test to measure quality on several repositories. And it can be used to detect if we introduce quality-degradation features.

**Comment by EgorBu** [comment]
List of repos:
```
https://github.com/vuejs/vue
https://github.com/vuejs/vue-cli
https://github.com/vuejs/vuex
https://github.com/GoogleChromeLabs/carlo
https://github.com/30-seconds/30-seconds-of-code  - quite small files
https://github.com/facebook/react
https://github.com/facebook/create-react-app
https://github.com/facebook/react-native
https://github.com/freeCodeCamp/freeCodeCamp
https://github.com/segmentio/evergreen
https://github.com/laravel/telescope
https://github.com/nodejs/node
https://github.com/nodejs/citgm
https://github.com/storybooks/storybook
https://github.com/axios/axios
https://github.com/jquery/jquery
https://github.com/atom/atom
https://github.com/webpack/webpack
https://github.com/reduxjs/redux
https://github.com/expressjs/express
https://github.com/meteor/meteor
```

**Comment by EgorBu** [comment]
| repo               |   prec |   recall |   f1 |   support |   n_rules |   avg_len |
|:-------------------|-------:|---------:|-----:|----------:|----------:|----------:|
| 30-seconds-of-code |   0.97 |     0.97 | 0.97 |   55716   |    393    |      7.2  |
| axios              |   0.9  |     0.93 | 0.91 |   32789   |     44    |      6    |
| carlo              |   0.82 |     0.88 | 0.85 |   15697   |    283    |      7.3  |
| citgm              |   0.95 |     0.95 | 0.95 |   25293   |     37    |      5.4  |
| create-react-app   |   0.94 |     0.95 | 0.94 |   21585   |    708    |      7.6  |
| express            |   0.91 |     0.93 | 0.92 |   94629   |    482    |      7.4  |
| redux              |   0.9  |     0.93 | 0.91 |   42759   |    711    |      7.7  |
| telescope          |   0.91 |     0.94 | 0.92 |    1684   |    136    |      5.7  |
| vue-cli            |   0.94 |     0.95 | 0.94 |    6851   |    216    |      6.7  |
| Weighted average   |   0.92 |     0.94 | 0.93 |           |           |           |
| Average            |   0.92 |     0.94 | 0.92 |   33000.3 |    334.44 |      6.78 |

**Comment by vmarkovtsev** [comment]
This is fixed by #326
